library/CHART-TEMPLATE/tests/common_library_lowdatamode_test.yaml

```diff
@@ -0,0 +1,82 @@
+suite: test lowdatamode helper
+templates:
+  - templates/example-cm-lowdatamode.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  - it: Is false by default
+    asserts:
```

Comment:
maybe setting lowDataMode to null here , to check for :boom nil pointers and making the test independent of the values.yaml ?

Reply:
Good point.

Solved with the last force push that rebases.